vhdbuilder/packer/components.json

```diff
@@ -132,7 +132,8 @@
       "downloadURL": "mcr.microsoft.com/azuremonitor/containerinsights/ciprod/prometheus-collector/images:*",
       "amd64OnlyVersions": [
         "5.1.0-main-09-23-2022-df3e2703",
```

Comment:
is this one still in use? "5.1.0-main-09-23-2022-df3e2703",
if not may remove it from the vhd.

Reply:
Yes. Second one is not fully rolled out. So we will keep all 3. Next release will remove first.